app/assets/v2/js/amounts.js

```diff
@@ -29,7 +29,10 @@ var get_rates_estimate = function(usd_amount) {
     rates_addon.push('' + hours + ' hrs at $' + rate + '/hr');
   }
   rates_addon = rates_addon.join(', ');
-  return rates_addon;
+
+  var help_addon = ' <a href="https://medium.com/gitcoin/tutorial-how-to-price-work-on-gitcoin-49bafcdd201e">[Read our pricing guide]</a>';
```

Comment:
Since it is an external link, I think we should put a `target=_blank`. I will push if everyone agrees with me. @mbeacom @thelostone-mc